app/screens/home/recent_mentions/recent_mentions.tsx

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useIsFocused, useRoute} from '@react-navigation/native';
+import React, {useCallback, useState, useEffect, useMemo} from 'react';
+import {useIntl} from 'react-intl';
+import {StyleSheet, View, ActivityIndicator, FlatList} from 'react-native';
+import Animated, {useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+import {SafeAreaView, Edge} from 'react-native-safe-area-context';
+
+import {getRecentMentions} from '@actions/remote/search';
+import DateSeparator from '@app/components/post_list/date_separator';
+import {UserModel} from '@app/database/models/server';
+import {getDateForDateLine, isDateLine, selectOrderedPosts} from '@app/utils/post_list';
```

Comment:
remove the app prefix

```suggestion
import DateSeparator from '@components/post_list/date_separator';
import {UserModel} from '@database/models/server';
import {getDateForDateLine, isDateLine, selectOrderedPosts} from '@utils/post_list';
```

this will need sorting